lib/MQTTPlateRecorder.js

```diff
@@ -172,14 +183,16 @@ export default class MQTTPlateRecorder extends PlateRecorder {
 				`${this.baseTopic}/${source.name}/plate`,
 				result.plate
 			);
-			this.#client.publish(
```

Comment:
Instead of lumping image & roi together, can you either only focus on the image or also provide support for "publishRoi"